app/client/components/FormRendererCss.ts

```diff
@@ -81,7 +82,11 @@ export const submitButton = styled('div', `
   justify-content: center;
   align-items: center;
 
-  & input[type="submit"] {
+  & :is(input[type="submit"], button:not([type]), button[type="submit"]) {
```

Comment:
Maybe `input[type="submit"]` is overkill now?

(Also below)

Reply:
this was me trying to have "future proof" css and deal with any possible submit button case but yeah maybe this is not that necessary and I can just write `button[type="submit"]`